app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DishesController < ApplicationController
+  def show
+    @dish = Dish.find(params[:id])
+    @ingredients = @dish.ingredients
```

Comment:
This isn't wrong, but in my personal opinion, I try to limit the number of ivars I send to the view. So in this case, I would only send `@dish` and then I can access @dish.ingredients from the view. I'm sure there are varying opinions on this. ")